src/Resolvers/Collections.ts

```diff
@@ -4,7 +4,7 @@ import { Collection } from "../Entities/Collection"
 
 @Resolver(of => Collection)
 export class CollectionsResolver {
-  private readonly repository = getMongoRepository(Collection)
+  protected readonly repository = getMongoRepository(Collection)
```

Comment:
Do we actually need this change?

Reply:
Actually no, we don't need this anymore as it was meant to expose the `repository` to a child class for mocking purposes